integration-authorization/src/main/java/org/symphonyoss/integration/authorization/oauth/v1/OAuth1HttpRequestException.java

```diff
@@ -3,6 +3,8 @@
 import org.symphonyoss.integration.exception.IntegrationException;
 
 /**
+ * Exception used to inform runtime exceptions regarding OAuth1HttpRequestException process.
```

Comment:
Checked exception to report HTTP error response during the API call to an external system using OAuth1 protocol.